reviewer_readme.md

```diff
@@ -1,46 +1,116 @@
+# Introduction
+
 ## Reviewing the Intel Android Hub
-So you've been chosen to review article submissions from Github for this funky fresh blog. I'll guide you through the things you need to do, in order to accept the article with confidence.
 
-## Setup
-1. Follow getting started instructions in the main project readme.md to get the basic project running
-2. Follow the rest of these instructions to determine if you'll accept or decline the article submission
+You've been chosen to review article submissions from Github for Intel's Android Hub. This guide will help you through the things you need to do, in order to accept the article with confidence.
+
+--------
+
+## Process Overview
+
+![process](https://drive.google.com/uc?id=0BwL_X3dS1kN_QVNwYXRydVR6VGs)
+
+--------
+
+### Step 1: Review Open Pull Requests
+
+Open the list of article submissions: https://github.com/intel-android/androidhub/pulls
+
+![pull requests](https://drive.google.com/uc?id=0BwL_X3dS1kN_Q3hRRFVGS2VMWWc)
+
+All submissions history is preserved and openly viewable. Clicking on a pull request (PR) will open that PR's details. Also appearing next to the title of the pull request is either a green checkmark or a red X indicating whether tests have passed.
+
+The PR details show you the files changed by the author, the author's commentary, and the pass/fail result of the build test. Review the files changed and follow these simple guidelines:
+
+##### Acceptable Changes:
```

Comment:
When we say “changes”, can we show a screenshot? Lets show them how they view/identify what has actually been changed